### PR TITLE
(feat): create routing for menu

### PR DESCRIPTION
**Pull request by parinpond**
Create router for menu
1.home
2.contact
3.product
4.about

### CODE REVIEW

File: src/app/app.module.ts [hunk 1]
```diff
@@ -7,6 +7,7 @@ import { HomeComponent } from './home/home.component';
 import { AboutComponent } from './about/about.component';
 import { ContactComponent } from './contact/contact.component';
 import { ProductComponent } from './product/product.component';
+import { RouterModule } from '@angular/router';
```
Comment (chalalaz) [inline src/app/app.module.ts:10]: please check the syntax

File: src/app/app.module.ts [hunk 2]
```diff
@@ -18,7 +19,13 @@ import { ProductComponent } from './product/product.component';
     ProductComponent
   ],
   imports: [
-    BrowserModule
+    BrowserModule,
```
Comment (chalalaz) [inline src/app/app.module.ts:21]: add more commend

File: src/app/nav/nav.component.html [hunk 3]
```diff
@@ -1,6 +1,61 @@
-<div>
-<a class="btn bth-primary">Home</a>
-<a class="btn bth-primary">About</a>
-<a class="btn bth-primary">Product</a>
-<a class="btn bth-primary">Contact</a>
+<nav class="navbar navbar-expand-sm navbar-dark bg-secondary">
+    <a class="navbar-brand" href="#">Navbar</a>
+    <button class="navbar-toggler d-lg-none" type="button" data-toggle="collapse" data-target="#collapsibleNavId" aria-controls="collapsibleNavId"
+        aria-expanded="false" aria-label="Toggle navigation"></button>
```
Comment (chalalaz) [inline src/app/nav/nav.component.html:4]: google ma chai mai

**Review by chalalaz — COMMENTED** [review]
please check

File: src/app/app.module.ts (hunk 1 above)
Comment (parinpond) [reply src/app/app.module.ts:10]: thank you. xxx

Comment (parinpond) [reply src/app/app.module.ts:10]: Please check again.
commit 17f55c4c1cfe161aaed935070d335666a4f1932d

**Pull request merged**